packages/overture-schema-addresses-theme/src/overture/schema/addresses/address/models.py

```diff
@@ -47,7 +48,9 @@ class Address(Feature[Literal["addresses"], Literal["address"]]):
 
     # Core
     geometry: Annotated[
-        Geometry, GeometryTypeConstraint("Point"), Field(description="Geometry (Point)")
+        Geometry,
+        GeometryTypeConstraint(GeometryType.POINT),
```

Comment:
Looking at this makes me think that `Geometry` should be generic over `GeometryType` instead of using an annotation.